modules/network-observability-flows-format.adoc

```diff
@@ -387,6 +414,48 @@ The "Cardinality" column contains information about the implied metric cardinali
 | no
 | avoid
 | timereceived
+| `Udns`
+| string[]
+| List of User Defined Networks
+| `udns`
+| no
+| null
+| n/a
+| `XlatDstAddr`
+| string
+| packet translation dst address
```

Comment:
Can dst and src be expanded to Source and Destination for uniformity? I see all other fields have Source and Destination instead of shorthands

Reply:
I added that here, and yep I agree it should be spelled out and a capital Packet https://github.com/netobserv/network-observability-operator/pull/1071/files#r1946811199